.drone.yml

```diff
@@ -1234,11 +1234,15 @@ steps:
     | jq .head.repo.fork)
   - if [ "$is_fork" != false ]; then return 1; fi
   - git clone "https://$${GITHUB_TOKEN}@github.com/grafana/grafana-enterprise.git"
-    grafana-enterprise
-  - cd grafana-enterprise
+    ../grafana-enterprise
+  - cd ../grafana-enterprise
```

Comment:
Maybe a stupid question - aren't this line and the line above equivalent?

Reply:
I think this is because I switched to using `enterprise_setup_step` instead of `clone_enterprise_step_pr`